DietPi-Software | Remot3.it: Fix install due to changed GitHub repo

MichaIng opened the pull request:
**Status**: Testing
- [x] Changelog
- [x] Online docs. Installer command updated to `connectd_installer`: https://dietpi.com/phpbb/viewtopic.php?p=188#p188

**Reference**: https://github.com/Fourdee/DietPi/issues/2551

**Testing**:
- [x] Stretch VM
- [x] Buster VM
- [x] Jessie VM

**Commit list/description**:
+ DietPi-Software | Remot3.it: Fix install due to changed GitHub repo. Install latest available version via: api.github.com
+ DietPi-Software | Remot3.it: Removed <=v150 uninstall code due to v160 update freeze
+ CHANGELOG | Remot3.it: Resolved an issue where install failed due to Git repo changes

MichaIng committed:
v6.22

+ DietPi-Software | Remot3.it: Fix install due to changed GitHub repo. Install latest available version via: api.github.com

MichaIng committed:
v6.22

+ DietPi-Software | Remot3.it: Do not check final download URL, which expectedly fails

Fourdee commented:
@MichaIng 

Thanks Micha, looks good 👍 

Please merge when changelog is updated.

MichaIng committed:
v6.22

+ CHANGELOG | Remot3.it: Resolved an issue where install failed due to Git repo changes

Fourdee commented:
@MichaIng 

Thanks 👍 

🈯️ ARMv7/8 testing passed.
